diff --git a/src/ldm-modules/ldm_common.c b/src/ldm-modules/ldm_common.c
index 7be2fd0..d8e021a 100644
--- a/src/ldm-modules/ldm_common.c
+++ b/src/ldm-modules/ldm_common.c
@@ -40,7 +40,7 @@ int mnttab_foreach(mntent_cb_t cb_func, void *cb_data)
     struct mntent *p_mnt;
     FILE          *fp;
     int            rc = 0;
-    char           mnt_buff[PAGE_SIZE];
+    char           mnt_buff[getpagesize()];
 
     assert(cb_func != NULL);
 
