diff --git a/src/request.c b/src/request.c
index 2c2a384..77b1319 100644
--- a/src/request.c
+++ b/src/request.c
@@ -907,8 +907,8 @@ static S3Status setup_curl(Request *request,
     // less than 1K per second for more than 15 seconds.
     // xxx todo - make these configurable
     // xxx todo - allow configurable max send and receive speed
-    curl_easy_setopt_safe(CURLOPT_LOW_SPEED_LIMIT, 1024);
-    curl_easy_setopt_safe(CURLOPT_LOW_SPEED_TIME, 15);
+    //curl_easy_setopt_safe(CURLOPT_LOW_SPEED_LIMIT, 1024);
+    //curl_easy_setopt_safe(CURLOPT_LOW_SPEED_TIME, 15);
 
     // Append standard headers
 #define append_standard_header(fieldName)                               \